<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>8051 Programmer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <item>
       <layout class="QHBoxLayout" name="connLayout">
        <item>
         <widget class="QComboBox" name="serialCombo"/>
        </item>
        <item>
         <widget class="QPushButton" name="serialConnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enableLVP">
        <property name="text">
         <string>Enable LVP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopLVP">
        <property name="text">
         <string>End LVP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downloadButton">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readButton">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eraseButton">
        <property name="text">
         <string>Erase</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="idButton">
        <property name="text">
         <string>Identify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearLogButton">
        <property name="text">
         <string>Clear Log</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="serialLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="editLayout">
      <item>
       <widget class="QHexEdit" name="hexEdit" native="true"/>
      </item>
      <item>
       <widget class="QHexEdit" name="confEdit" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApp">
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionOpen_Program"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuApp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Program">
   <property name="text">
    <string>Open Program</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header location="global">qhexedit/qhexedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen_Program</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
